docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,241 @@
+r"""
+Calculating the cell temperature for floating PV
+================================================
+
+This example uses the PVSyst temperature model to calculate the
+cell temperature for floating photovoltaic (FPV) systems.
+
+One of the primary benefits attributed to FPV systems
+is lower operating temperatures, which are expected to increase the
+operating efficiency. In general, the temperature at which a photovoltaic
+module operates is influenced by various factors including solar radiation,
+ambient temperature, wind speed and direction, and the characteristics of the
+cell and module materials, as well as the mounting structure. Both radiative
+and convective heat transfers play roles in determining the module's
+temperature.
+
+A popular model for calculating the PV cell temperature is the
+empirical heat loss factor model suggested by Faiman. A modified version of
```

Comment:
Add a citation for the Faiman model here?
http://dx.doi.org/10.1002/pip.813?

and/or possibly reference the pvlib function, e.g.
```suggestion
empirical heat loss factor model suggested by Faiman (:py:func:`pvlib.temperature.faiman`). A modified version of
```